test/topology_custom/test_remove_rpc_client_with_pending_requests.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (C) 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+import asyncio
+import logging
+import time
+
+import pytest
+from cassandra.query import SimpleStatement, ConsistencyLevel
+
+from test.pylib.manager_client import ManagerClient
+from test.pylib.random_tables import RandomTables, Column, TextType
+from test.pylib.util import wait_for_cql_and_get_hosts
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.mark.asyncio
+async def test_remove_rpc_client_with_pending_requests(request, manager: ManagerClient) -> None:
+    # Regression test for #17445
+
+    logger.info("starting first node")
+    servers = [await manager.server_add()]
+
+    logger.info("starting second node")
+    servers += [await manager.server_add()]
```

Comment:
Why not add them concurrently with `servers_add`?

Reply:
no reason, changed to `servers_add`